phonebuddies/PhoneBuddyManager.py

```diff
@@ -103,6 +103,33 @@ def admin_buddy_pairs(self):
         return [ADMIN_EMAILS[:2]]
 
 
+    def send_admins_without_buddy_emails(self, buddy_pairs):
+        buddy_emails_list=[]
+        admins=ADMIN_EMAILS
+        admins_without_buddies=[]
```

Comment:
Incredibly nitpick-y, but the preferred style in this repo is to put a space before and after the `=`